presto-spi/src/main/java/com/facebook/presto/spi/block/RunLengthEncodedBlock.java

```diff
@@ -145,6 +145,12 @@ public long getRegionSizeInBytes(int position, int length)
         return value.getSizeInBytes();
     }
 
+    @Override
+    public long getPositionsSizeInBytes(boolean[] positions)
+    {
+        return value.getSizeInBytes();
```

Comment:
I think you need to see if positions is not all `false`.

Reply:
So if it's all false then return 0? That makes sense. In general the concept of getting size of this block is confusing to me.

Reply:
Interestingly, if I return 0 when all positions are false, there are test failures in `TestRunLengthEncodedBlock`. The `getRegionSizeInBytes` also doesn't check `length > 0` so at least without the check the two function behave the same way. It makes sense to return 0 if all positions are false or if length is 0 though. I don't know whether the test was wrong or what. For now I removed the special check so test would pass as is.